src/lyric.js

```diff
@@ -46,7 +46,25 @@ class LRCLIBProvider { // Ref: https://lrclib.net/docs
     }
 }
 
-const Providers = [NeteaseProvider, LRCLIBProvider];
+class NeteaseTransProvider {
```

Comment:
Could you [try](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends) `extends NeteaseProvider`?